siddpubs-journal.bib

```diff
@@ -28,10 +28,10 @@ @article{chen2019trust
 }
 
 @article{gammell2019bitstar,
```

Comment:
Do we want to update the key to `gammell2020bitstar`?

Reply:
Done!